src/marten/db/query/set.cr

```diff
@@ -325,6 +325,41 @@ module Marten
           add_query_node(query_node)
         end
 
+        # Returns if a specific model exists in the query set.
+        def exists?(value : M)
```

Comment:
I think the name of this specific method only could've remained `#includes?` in order to fix the initial issue. I'll fix this after merging this.